### DISCUSSION TITLE
How to update HA with new Wyze password?

### DISCUSSION THREAD

**Discussion by grein002** [SecKatie/ha-wyzeapi#379] [Q&A]
I've just changed my Wyze password due to the old one being on a compromised list. Is there a simple way for me to update the password within HA?

**Comment by JoeSchubert** [comment]
I hate to alarm you, especially ~9 months later.

But if your Wyze account was compromised, you may need to contact them to manually invalidate all existing logins. I know of no way to do this through their app, and changing your password does not invalidate any existing logins as far as I can tell. Once something/someone has an access token and refresh token, they do not use your password (or MFA) to renew the tokens. Even if you had a compromised password, changed it and enabled MFA, an existing set of tokens will be able to renew indefinitely from what I have seen.
